FirebaseRemoteConfigSwift/Tests/SwiftAPI/PropertyWrapper.swift

```diff
@@ -0,0 +1,297 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import FirebaseRemoteConfig
+import FirebaseRemoteConfigSwift
+import XCTest
+
+#if compiler(>=5.5.2) && canImport(_Concurrency)
+  @available(iOS 14.0, macOS 11.0, macCatalyst 14.0, tvOS 14.0, watchOS 7.0, *)
+  class PropertyWrapperTests: APITestBase {
```

Comment:
nit: rename the filename to `PropertyWrapperTests.swift`

Fwiw, I've had difficulty renaming files properly in the past but one thing ive learned is to rename and then commit the rename by itself. Sometimes things get weird when you rename a file and have other changes. 

```
git mv \
  FirebaseRemoteConfigSwift/Tests/SwiftAPI/PropertyWrapper.swift \
  FirebaseRemoteConfigSwift/Tests/SwiftAPI/PropertyWrapperTests.swift
git commit -m "Rename PropertyWrapper to PropertyWrapperTests"
```

Just sharing in case you have ever  into similar issues with renaming!!

Reply:
Done.